easybuild/easyblocks/p/pybind11.py

```diff
@@ -91,13 +91,18 @@ def sanity_check_step(self):
         # don't add user site directory to sys.path (equivalent to python -s)
         env.setvar('PYTHONNOUSERSITE', '1', verbose=False)
         # Get python includes
-        fake_mod_data = self.load_fake_module(purge=True)
+        if not self.is_extension:
+            # for stand-alone Python package installations (not part of a bundle of extensions),
+            # we need to load the fake module file, otherwise the Python package being installed
+            # is not "in view", and we will overlook missing dependencies...
+            fake_mod_data = self.load_fake_module(purge=True)
         cmd = "%s -c 'import pybind11; print(pybind11.get_include())'" % self.python_cmd
```

Comment:
This part can only be done when we can load the function, we have to exclude all of it.

Reply:
in fact, excluding the `load_fake_module` only is precisely what is done in the PythonPackage EasyBlock, with a bunch of stuff between that line and the `clean_up_fake_module` line at the end

Reply:
hm, ok i guess the bundle then does the load before running through all the extensions, ok